Source/Foundation/bsfUtility/Reflection/BsIReflectable.cpp

```diff
@@ -35,10 +35,9 @@ namespace bs
 	RTTITypeBase* IReflectable::_getRTTIfromTypeId(UINT32 rttiTypeId)
 	{
 		Stack<RTTITypeBase*> todo;
-		Vector<RTTITypeBase*>& rootClasses = getDerivedClasses();
 
-		for(auto iter = rootClasses.begin(); iter != rootClasses.end(); ++iter)
-			todo.push(*iter);
+		for(const auto & item : getDerivedClasses())
```

Comment:
I prefer to keep the & next to the type, like so `const auto&`. I'm in the process of merging right now so I'll modify it on my end.